rbc/tests/heavydb/test_type_to_extarg.py

```diff
@@ -0,0 +1,148 @@
+    "int16*",
+    "int32*",
+    "int64*",
+    "float32*",
+    "float64*",
+    "bool",
+    "bool*",
+    "Cursor",
+    "void",
+    "GeoPoint",
+    "GeoLineString",
+    "GeoPolygon",
+    "GeoMultiPolygon",
+    "GeoMultiLineString",
+    "GeoMultiPoint",
+    "TextEncodingNone",
+    "TextEncodingDict",
+    "Timestamp",
+    "DayTimeInterval",
+    "YearMonthTimeInterval",
+    # Array
+    "Array<bool>",
+    "Array<int8_t>",
+    "Array<int16_t>",
+    "Array<int32_t>",
+    "Array<int64_t>",
+    "Array<float>",
+    "Array<double>",
+    "Array<TextEncodingDict>",
+    "Array<TextEncodingNone>",
+    "Array<GeoPoint>",
+    "Array<GeoMultiPoint>",
+    "Array<GeoLineString>",
+    "Array<GeoMultiLineString>",
+    "Array<GeoPolygon>",
+    "Array<GeoMultiPolygon>",
+    # Column
+    "Column<bool>",
+    "Column<int8_t>",
+    "Column<int16_t>",
+    "Column<int32_t>",
+    "Column<int64_t>",
+    "Column<float>",
+    "Column<double>",
+    "Column<TextEncodingDict>",
+    "Column<TextEncodingNone>",
+    "Column<Timestamp>",
+    "Column<GeoPoint>",
+    "Column<GeoMultiPoint>",
+    "Column<GeoLineString>",
+    "Column<GeoMultiLineString>",
+    "Column<GeoPolygon>",
+    "Column<GeoMultiPolygon>",
+    # ColumnList
+    "ColumnList<bool>",
+    "ColumnList<int8_t>",
+    "ColumnList<int16_t>",
+    "ColumnList<int32_t>",
+    "ColumnList<int64_t>",
+    "ColumnList<float>",
+    "ColumnList<double>",
+    "ColumnList<TextEncodingDict>",
+    "ColumnList<TextEncodingNone>",
+    "ColumnList<GeoPoint>",
+    "ColumnList<GeoMultiPoint>",
+    "ColumnList<GeoLineString>",
+    "ColumnList<GeoMultiLineString>",
+    "ColumnList<GeoPolygon>",
+    "ColumnList<GeoMultiPolygon>",
+    # ColumnArray
+    "Column<Array<bool>>",
+    "Column<Array<int8_t>>",
+    "Column<Array<int16_t>>",
+    "Column<Array<int32_t>>",
+    "Column<Array<int64_t>>",
+    "Column<Array<float>>",
+    "Column<Array<double>>",
+    "Column<Array<TextEncodingDict>>",
+    "Column<Array<TextEncodingNone>>",
+    "Column<Array<GeoPoint>>",
+    "Column<Array<GeoMultiPoint>>",
+    "Column<Array<GeoLineString>>",
+    "Column<Array<GeoMultiLineString>>",
+    "Column<Array<GeoPolygon>>",
+    "Column<Array<GeoMultiPolygon>>",
+    # ColumnListArray
+    "ColumnList<Array<bool>>",
+    "ColumnList<Array<int8_t>>",
+    "ColumnList<Array<int16_t>>",
+    "ColumnList<Array<int32_t>>",
+    "ColumnList<Array<int64_t>>",
+    "ColumnList<Array<float>>",
+    "ColumnList<Array<double>>",
+    "ColumnList<Array<TextEncodingDict>>",
+    "ColumnList<Array<TextEncodingNone>>",
+    "ColumnList<Array<GeoPoint>>",
+    "ColumnList<Array<GeoMultiPoint>>",
+    "ColumnList<Array<GeoLineString>>",
+    "ColumnList<Array<GeoMultiLineString>>",
+    "ColumnList<Array<GeoPolygon>>",
+    "ColumnList<Array<GeoMultiPolygon>>",
+)
+
+
+def test_len(heavydb):
+    msg = 'variable "all_types" requires update'
+    assert len(heavydb._get_ext_arguments_map()) == len(all_types), msg
+
```

Comment:
I wonder, should we also test the exceptions from user-specified ill-formed types names? For example, when they specify `Array<complex>` in the heavydb signature, will the error message be informative enough for users to not struggle with the unsupported type?